packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -483,9 +484,19 @@ exports.createNodesForContentType = ({
               )
             })
 
+          const deprecationReason = `This field got removed due to performance reasons. Please use the new "raw" and "references" fields. See: https://www.gatsbyjs.com/plugins/gatsby-source-contentful/#contentful-rich-text`
```

Comment:
```suggestion
          const deprecationReason = `This field was removed in 
gatsby-source-contentful v4. Please use the new "raw" and "references" fields. See: https://www.gatsbyjs.com/plugins/gatsby-source-contentful/#contentful-rich-text`
```

Works for me. If you want you can take a tiny adjustment, but no hard feelings he. :)